src/packages/@ncigdc/utils/wrapSvg.js

```diff
@@ -177,6 +179,12 @@ export const wrapSvg: TWrapSvg = ({
         )}
       </g>
       ${afterObject.html}
+      <foreignobject  class="node" x="0" y="${sum([
```

Comment:
Is there a way to add this only when there is a legend displayed? (when there's a legends prop)